cmd/atlas-import/cli.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+func generateBaseConfig(cmd *cobra.Command) AtlasImportConfig {
+	baseConfig := AtlasImportConfig{
```

Comment:
I quite strongly think that this should share config & API keys with the atlascli project